client/landing/jetpack-cloud/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import config from '../../config';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import initJetpackCloudRoutes from './routes';
+
+/**
+ * Style dependencies
+ */
+import 'components/environment-badge/style.scss';
+import 'layout/style.scss';
+import 'assets/stylesheets/jetpack-cloud.scss';
+
+window.AppBoot = () => {
+	if ( ! config.isEnabled( 'jetpack-cloud' ) ) {
```

Comment:
I am not sure this works as expected. 
At least I wasn't able to show me jetpack cloud when I set 'jetpack-cloud': true, in the `development.conf` file. 

How can I test this? 
If this doesn't work for now can we just remove it?

Reply:
As discussed via Slack, the idea is that if you try to access `/jetpack-cloud` on any other env (production) for example, it will redirect you to `/`. And right now we've enabled this only for the dev environment.